exonum/src/api/websocket.rs

```diff
@@ -33,20 +137,33 @@ pub(crate) struct Message(pub String);
 #[rtype(usize)]
 pub(crate) struct Subscribe {
     pub address: Recipient<Message>,
+    pub sub_types: Vec<SubscriptionType>,
```

Comment:
Such name isn't so clear because it can be confused with subtypes.

Reply:
Done (renamed to `subscriptions`)